app/models/reaction.rb

```diff
@@ -80,7 +80,7 @@ def touch_user
   end
 
   def update_reactable
```

Comment:
This is called in an `after_save` callback which means the reaction will not be persisted to the database yet. We need to move this to an `after_commit` callback